janus_server/src/aggregator/aggregation_job_creator.rs

```diff
@@ -220,6 +220,13 @@ impl<C: Clock + 'static> AggregationJobCreator<C> {
                     .await
             }
 
+            crate::task::VdafInstance::Real(janus_core::task::VdafInstance::Prio3Aes128CountVec { .. }) => {
```

Comment:
not caused by this PR: consider importing `crate::task::VdafInstance` so the match arms can be written as `VdafInstance::Real(...)` as they are in other instances of this kind of match.